restapi/src/main/java/io/stargate/web/docsapi/dao/Paginator.java

```diff
@@ -29,6 +29,27 @@ public Paginator(String pageState, int pageSize) {
     }
   }
 
+  /**
+   * Utility to make the external paging state from docs, without side effects in this {@link
+   * Paginator}.
+   *
+   * @param docs document
+   * @return String to serve to external
+   */
+  public String makeExternalPagingState(List<RawDocument> docs) {
```

Comment:
Could we make this method `static`? I'd like to minimize the possibility of confusion about how we track and report paging state. I mean in this case we do not change the `Paginator`'s internal state.

Reply:
Done.